src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/GoalEntityBuilder.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity
+
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.GoalCategory.PERSONAL_DEVELOPMENT
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.GoalStatus.ACTIVE
+import java.time.Instant
+import java.time.LocalDate
+import java.util.UUID
+
+fun aValidGoalEntity(
+  id: UUID? = UUID.randomUUID(),
+  reference: UUID = UUID.randomUUID(),
+  title: String = "Improve communication skills",
+  reviewDate: LocalDate = LocalDate.now().plusMonths(6),
+  category: GoalCategory = PERSONAL_DEVELOPMENT,
+  status: GoalStatus = ACTIVE,
+  notes: String? = "Chris would like to improve his listening skills, not just his verbal communication",
+  steps: MutableList<StepEntity> = mutableListOf(aValidStepEntity()),
+  createdAt: Instant? = Instant.now(),
+  createdBy: String? = "a.user.id",
+  updatedAt: Instant? = Instant.now(),
+  updatedBy: String? = "another.user.id",
```

Comment:
Minor - might it be be worth updating equivalent ones in other builders, so we can change them all in one go later?

Reply:
I think I'm introducing something along those lines in my next PR, so hold that thought .... 😁